packages/merlin-server/PKGBUILD

```diff
@@ -0,0 +1,45 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname='merlin-server'
+pkgver='0.6.0'
```

Comment:
There's no need for quotes here. It's actually a bad thing, it might lead to invalid names being interpreted by bash.

(Note this is just a small nitpick)